ouroboros-consensus/src/Ouroboros/Consensus/HardFork/Combinator/Ledger/Query.hs

```diff
@@ -309,14 +326,19 @@ answerQueryAnytime HardForkLedgerConfig{..} =
 -------------------------------------------------------------------------------}
 
 data QueryHardFork xs result where
-  GetInterpreter :: QueryHardFork xs (History.Interpreter xs)
-  GetCurrentEra  :: QueryHardFork xs (EraIndex xs)
+  GetInterpreter  :: QueryHardFork xs (History.Interpreter xs)
+  GetCurrentEra   :: QueryHardFork xs (EraIndex xs)
+  GetLedgerCfg    :: QueryHardFork xs (HardForkLedgerConfig xs)
 
```

Comment:
Please add a comment here that refers to the excellent comment at the end of `Query`.